src/coreclr/vm/stublink.cpp

```diff
@@ -2110,6 +2110,10 @@ Stub* Stub::NewStub(PTR_VOID pCode, DWORD flags)
         size += numCodeBytes;
     }
 
+    // Add pointer to store the target MethodDesc.
+    if (flags & NEWSTUB_FL_INSTANTIATING_METHOD)
```

Comment:
What do you think of putting this in a statically defined field like this:
```C++
class Stub
{
    ULONG m_refcount;
    UINT m_numCodeBytesAndFlagBits; // high 5 bits are flags (currently in m_patchOffset), remaining bits are size
    union
    {
        ULONG m_patchOffset; 
        PTR_MethodDesc m_pInstantiatedMethod;
    }

    UINT GetPatchOffset() { IsInstantiatingStub() : 0 ? m_patchOffset; }
    PTR_MethodDesc GetInstantedMethodDesc() { _ASSERTE(IsInstantiatingStub()); return m_pInstantiatedMethod; }
}
```

This keeps Stub the same size it is now (12 or 16 bytes), avoids the more complex dynamic size computation, and should make it a little easier when viewing Stub in the debugger to determine that MD pointer.

Reply:
This seems to make sense. I'll do this.